### PR TITLE
command manifest

### PR DESCRIPTION
**Pull request by artem-starovoitov**
# PR Details

https://trello.com/c/4WpZUOdi/62-add-manifest-command

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: client/lib/agile/assets/manifest.txt [hunk 1]
```diff
@@ -0,0 +1,52 @@
+Values:
```
Comment (nieisnuje) [inline client/lib/agile/assets/manifest.txt:1]: Manifesto:
https://agilemanifesto.org/

File: client/lib/agile/commands/manifest.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+module Agile
+  class CLI < Thor
+    desc Rainbow("manifest").cornflower, Rainbow("Shows manifest").darkgoldenrod
```
Comment (nieisnuje) [inline client/lib/agile/commands/manifest.rb:3]: manifesto not manifest

**Review by nieisnuje — CHANGES_REQUESTED** [review]

File: client/lib/agile/assets/manifest.txt (hunk 1 above)
Comment (Berendor) [inline client/lib/agile/assets/manifest.txt:1]:
```suggestion
Manifesto for Agile Software Development
```

File: client/lib/agile/commands/manifest.rb (hunk 2 above)
Comment (Berendor) [inline client/lib/agile/commands/manifest.rb:3]:
```suggestion
    desc Rainbow("manifesto").cornflower, Rainbow("Shows manifest").darkgoldenrod
```

**Pull request closed without merging**